### PR TITLE
Changes to the javascript-sdk-migration branch

### PR DESCRIPTION
**Pull request by tomijaga**
#617 
I replaced the try-catch block with the `Account.isValidPair` static method

### CODE REVIEW

File: src/renderer/containers/Validator/AddValidatorSigningKeysModal/index.tsx [hunk 1]
```diff
@@ -54,12 +54,8 @@ const AddValidatorSigningKeysModal: FC<ComponentProps> = ({close}) => {
...
 
   const checkPrivateSigningKey = (publicKey: string, privateKey: string): boolean => {
-    try {
-      const {accountNumberHex} = new Account(privateKey);
-      return accountNumberHex === publicKey;
-    } catch (error) {
-      return false;
-    }
+    const {accountNumberHex} = new Account(privateKey);
+    return Account.isValidPair(accountNumberHex, publicKey);
```
Comment (sno2) [inline src/renderer/containers/Validator/AddValidatorSigningKeysModal/index.tsx:58]: You should be able to just do this:

```ts
const checkPrivateSigningKey = (publicKey: string, privateKey: string): boolean => 
  Account.isValidPair(privateKey, publicKey)
```

Note: the private key (signing key) goes second and the public key (account number) goes first.

**Review by sno2 — CHANGES_REQUESTED** [review]

**Comment by tomijaga** [comment]
@sno2 I've made all the required changes

**Comment by sno2** [comment]
> @sno2 I've made all the required changes

Nice, but I'm just a contributor so I can't merge ;)

**Comment by tomijaga** [comment]
@sno2 Thanks for the review

**Comment by tomijaga** [comment]
@zinoadidi I've made some changes to the migration branch can you please merge them.

**Pull request merged**